minitwit_frontend/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /minitwit_frontend
+WORKDIR /minitwit_frontend
+
+COPY requirements.txt /minitwit_frontend/
+EXPOSE 8000
+
+RUN pip install -r requirements.txt
+
+COPY . /minitwit_frontend/
+COPY scripts/entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
+
+# RUN mkdir -p /vol/web/media
+# RUN mkdir -p /vol/static
+# RUN adduser user
+# RUN chown -R user:user /vol
+# RUN chmod -R 755 /vol/web
+# USER user
```

Comment:
Any reason for keeping this multiple line comment of code?
Only line 18 seems to differ from line 26

Reply:
Not at all!

Reply:
But the view here looks outdated: https://github.com/notaduck/DevOps_ITU/edit/dockerize_mono/minitwit_frontend/Dockerfile?pr=%2Fnotaduck%2FDevOps_ITU%2Fpull%2F27

It should have been uncommented.